mobile/big.go

```diff
@@ -35,6 +35,16 @@ func NewBigInt(x int64) *BigInt {
 	return &BigInt{big.NewInt(x)}
 }
 
+// NewBigIntFromString allocates and returns a new BigInt set to x
+// interpreted in the provided base.
+func NewBigIntFromString(x string, base int) *BigInt {
+	b, success := big.NewInt(0).SetString(x, base)
```

Comment:
```suggestion
	b, success := (new big.Int).SetString(x, base)
```

Although in general, wouldn't it be cleaner API to return `(*Bigint, ok)` (same as `SetString`) instead of returning a `nil`?

So just 
```golang

func NewBigIntFromString(x string, base int) (*BigInt, bool) {
	if b, ok := (new big.Int).SetString(x, base); ok{
		return &BigInt{b}, true
	}
	return nil, false
}
```

Reply:
Yeah that would be nicer, but the cool thing if I do it like this is, that gomobile will overload the constructor. 
You can then use `BigInt b = new BigInt("1234", 10);` and `BigInt b = new BigInt(1234);` in java

Reply:
Ok then!